### PR TITLE
Snap e and t update

### PR DESCRIPTION
**Pull request by IlseFerris**
This function formats a string (y) being entered into a case note. This function retains the functionality from the old function write_new_line_in_case_note(x) but enables for line headers (bullet) and indentation.  The header must be stored in quotation marks. The function automatically places a colon ( : ) at the end of the header. 
The function also checks to determine if the case note needs to go further than 4 pages and will modify the case note header to read (1 or 2) and create a new case note with header and (2 of 2) if necessary.

Resolves issue #819

### CODE REVIEW

File: Script Files/MEMOS/MEMOS - SNAP E AND T LETTER.vbs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+worker_county_code = "x127"
```
Comment (theVKC) [inline Script Files/MEMOS/MEMOS - SNAP E AND T LETTER.vbs]: This won't work for 86 of the counties, @IlseFerris. :P

**Comment by theVKC** [comment]
Closing this because of `worker_county_code`, @IlseFerris. Please fix that and resubmit. :)

**Comment by IlseFerris** [comment]
Dang it, I knew I'd forget!  On it.

File: Script Files/MEMOS/MEMOS - SNAP E AND T LETTER.vbs (hunk 1 above)
Comment (IlseFerris) [inline Script Files/MEMOS/MEMOS - SNAP E AND T LETTER.vbs]: Wait, what...

**Pull request closed without merging**